workbench/icons/Makefile.am

```diff
@@ -0,0 +1,20 @@
+icondir = $(datadir)/icons/hicolor/16x16/apps
```

Comment:
I'm not sure I understand. If you mean the icons when loaded in to Geany, I believe the functions that load icons allow to specify the size (ex. `GTK_ICON_SIZE_MENU`, etc).